packages/atomic/src/components/insight/atomic-insight-interface/atomic-insight-interface.mdx

```diff
@@ -0,0 +1,13 @@
+import { Meta } from '@storybook/blocks';
```

Comment:
Given that atomic-insight is tagged internal, we could probably do away with the mdx altogether, but I'm including it in the draft.